translator/cmdutil/userutil_linux.go

```diff
@@ -6,29 +6,108 @@
 package cmdutil
 
 import (
+	"bufio"
+	"bytes"
 	"fmt"
+	"golang.org/x/sys/unix"
+	"io"
 	"log"
 	"os"
-	"syscall"
+	"os/user"
+	"strconv"
+	"strings"
+)
 
-	"golang.org/x/sys/unix"
+type ExecUser struct {
+	Uid  int
+	Gid  int
+	Home string
+	Gids []int
+}
 
-	"github.com/opencontainers/runc/libcontainer/system"
-	"github.com/opencontainers/runc/libcontainer/user"
-)
+func getGroupIds(u *user.User) ([]int, error) {
```

Comment:
I skimmed it and it looks good but can how hard would it be to include a test for this code?